abel/hansenlaw.py

```diff
@@ -170,9 +170,9 @@ def igammagt(Nm, lam, n):
     # center pixel column
     AIM[:, 0] = AIM[:, 1]
 
-    # for some reason shift by 1/2 pixel aligns? - Fix me!
+    # for some reason shift by -0.3 pixel aligns? - Fix me!
     if direction == 'inverse':
-        AIM = shift(AIM, (0, -1/2))
+        AIM = shift(AIM, (0, -0.30))
```

Comment:
@DanHickstein thanks for the comments. I will work through them.

Yes, an only on the `hansenlaw` inverse Abel transform. This is still an open issue #159.